bin/pd-nagios

```diff
@@ -176,6 +185,18 @@ def parse_fields(fields):
         return {}
     return dict(f.split("=", 1) for f in fields)
 
+def parse_env_fields(details, env_fields):
+    if env_fields is None:
+        return
+    import os
+    for f in env_fields:
+        v = os.environ.get("NAGIOS_%s" % f)
+        if v is None:
+            v = os.environ.get("ICINGA_%s" % f)
+        if v is None:
```

Comment:
should we just do something like this instead?

```python
if v is not None:
  details[f] = v
```

Otherwise, I imagine we'll be creating events with some `"" => ""` details fields.